appveyor.yml

```diff
@@ -51,7 +51,7 @@ before_build:
 - cmd: set CPATH=%LIBELF_INCLUDE%;%CPATH%
 - cmd: set CPLUS_INCLUDE_PATH=%LIBELF_INCLUDE%;%BOOST_ROOT%;%CPLUS_INCLUDE_PATH%
 # set variables for MSVC
-- cmd: if "%build%"=="msvc"  call "%VS140COMNTOOLS%\..\..\VC\vcvarsall.bat" %PLATFORM%
+- cmd: if "%build%"=="msvc"  call "%VS153COMNTOOLS%\..\..\VC\vcvarsall.bat" %PLATFORM%
```

Comment:
I think you need to try VS150COMNTOOLS. Google knows nothing about VS153COMNTOOLS.